addon/adapters/rest.js

```diff
@@ -1245,6 +1246,14 @@ function ajaxError(adapter, payload, requestData, responseData) {
   return error;
 }
 
+// Adapter abort error to include any relevent info, e.g. request/response:
+// 'The adapter operation was aborted'
+function handleAbort(requestData, responseData) {
+  let msg = `Request failed: ${requestData.method} ${requestData.url} ${requestData.errorThrown || ''}`;
+  let errors = [{ title: 'Adapter Error', detail: msg.trim(), status: responseData.status }];
+  return new AbortError(errors);
```

Comment:
Having this error with the url, and error response if available would be much better. 

Currently we only use `new AbortError();`